src/Controller/Traits/LoginTrait.php

```diff
@@ -130,7 +130,7 @@ public function failedSocialLogin($exception, $data, $flash = false)
         }
         if ($flash) {
             $this->request->getSession()->delete(Configure::read('Users.Key.Session.social'));
-            $this->Flash->success(__d('CakeDC/Users', $msg), ['clear' => true]);
```

Comment:
Why this removed?

We should keep the domain based translation of the required messages

Reply:
I removed it because the content of this translation is a variable (and only a variable) and that doesn't seem to make sense to me (I might be wrong though).

If I try to extract i18n with the cake console task I get "Invalid marker content".

Reply:
Ah, you are right, it makes no sense to try to translate $msg twice, as it's already picking a domain translated string